extensions/wikia/AdEngine/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "app",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "css-loader": "^0.28.7",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "node-sass": "^4.6.0",
+    "sass-loader": "^6.0.6",
+    "style-loader": "^0.19.0",
+    "webpack": "^3.8.1"
+  },
+  "scripts": {
+    "build": "webpack -p",
+    "watch": "webpack -p -w"
+  },
+  "dependencies": {
+    "ad-engine": "git+ssh://git@github.com:Wikia/ad-engine.git",
+    "ad-products": "git+ssh://git@github.com:Wikia/ad-products.git"
```

Comment:
Shouldn't we set the version for this dependency?

Reply:
no :( 
```
import Context from 'ad-engine/src/services/context-service';
import SlotService from 'ad-engine/src/services/slot-service';
import TemplateService from 'ad-engine/src/services/template-service';
import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
```